fixham/internal/utils/files.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// CreateDir creates a directory if it does not exist.
+func CreateDir(dir string) error {
```

Comment:
I don't think we need a helper for this. We can just do `os.MkdirAll()` - it already handles skipping directories that already exist.